.github/workflows/main.yml

```diff
@@ -20,6 +20,7 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
+        python -m pip install git+https://github.com/mathurinm/celer.git
```

Comment:
why adding this, normally the code is compiled locally with `pip install .`

Reply:
This is to install ``celer`` for CI tests.
I wanted to release from running the test locally and expose the result directly.

Reply:
my bad